### PR TITLE
Create Microsoft.CodeCoverage package

### PR DESCRIPTION
**Pull request by smadala**
## Description
- Create Microsoft.CodeCoverage Nuget package.
[More details](https://microsoft.sharepoint.com/teams/DD_TestingTools/_layouts/OneNote.aspx?id=%2Fteams%2FDD_TestingTools%2FDocuments%2FUT%20CUIT%2Fdev15%2FVSTest%20%28core%29&wd=target%28Design.one%7C5F330F1B-7783-402D-9305-16093F37EA63%2FSpike%20for%20automatically%20discovery%20of%20TraceDataCollector%28TDC%5C%29%7C71E08665-1538-4C92-B78D-8619932C9931%2F%29)
## Related issue
#981

### CODE REVIEW

File: scripts/build.ps1 [hunk 1]
```diff
@@ -587,9 +602,32 @@ function Create-NugetPackages
         Set-ScriptFailedOnError
     }
 
+    Verify-Nuget-Packages $packageOutputDir
```
Comment (mayankbansal018) [inline scripts/build.ps1]: Add a comment to specify why this is needed

File: src/package/nuspec/Microsoft.CodeCoverage.nuspec [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <tags>vstest visual-studio unittest testplatform mstest microsoft test testing codecoverage code-coverage</tags>
+  </metadata>
+
+  <files>
+
+    <file src="Microsoft.CodeCoverage.props" target="build\netstandard1.0\" />
+    <file src="Microsoft.CodeCoverage\Microsoft.VisualStudio.TraceDataCollector.dll" target="build\netstandard1.0\" />
+    <file src="Microsoft.CodeCoverage\CodeCoverage\**\*.*"  exclude="**\*.pdb" target="build\netstandard1.0\" />
+    <file src="Microsoft.CodeCoverage\Microsoft.VisualStudio.CodeCoverage.Shim.dll" target="lib\netstandard1.0\" />
```
Comment (mayankbansal018) [inline src/package/nuspec/Microsoft.CodeCoverage.nuspec]: >CodeCoverage [](start = 61, length = 12)

Shim for Core/Full would need to be in separate .net frameworks (netcore1.0, net451)

File: src/package/sign/sign.proj [hunk 3]
```diff
@@ -279,6 +280,20 @@
       <BuildAssembliesToSign Include="$(ArtifactsBaseDirectory)Microsoft.TestPlatform.Build\netstandard1.3\%(ResxLang.Identity)\*.*" />
     </ItemGroup>
 
+    <!-- Sign Microsoft.CodeCoverage -->
+    <ItemGroup>
+      <CodeCoverageAssembliesToSign Include="$(ArtifactsBaseDirectory)Microsoft.CodeCoverage\Microsoft.VisualStudio.TraceDataCollector.dll" />
+      <CodeCoverageAssembliesToSign Include="$(ArtifactsBaseDirectory)Microsoft.CodeCoverage\%(ResxLang.Identity)\*.*" />
+      <CodeCoverageAssembliesToSign Include="$(ArtifactsBaseDirectory)Microsoft.CodeCoverage\Microsoft.VisualStudio.CodeCoverage.Shim.dll" />
```
Comment (mayankbansal018) [inline src/package/sign/sign.proj]: Add sign full clr shim as well

**Review by mayankbansal018 — CHANGES_REQUESTED** [review]
:clock1:

File: scripts/build.ps1 (hunk 1 above)
Comment (smadala) [reply scripts/build.ps1]: Done.

File: src/package/nuspec/Microsoft.CodeCoverage.nuspec (hunk 2 above)
Comment (smadala) [reply src/package/nuspec/Microsoft.CodeCoverage.nuspec]: Done.

File: src/package/sign/sign.proj (hunk 3 above)
Comment (smadala) [reply src/package/sign/sign.proj]: Done.

**Review by mayankbansal018 — APPROVED** [review]
:shipit:

**Pull request merged**